storage/src/main/java/tech/pegasys/artemis/storage/ChainStorageClient.java

```diff
@@ -59,32 +60,33 @@
   // Time
   private volatile UnsignedLong genesisTime;
 
-  public static ChainStorageClient memoryOnlyClient(final EventBus eventBus) {
-    final ChainStorageClient client =
-        new ChainStorageClient(eventBus, TransactionPrecommit.memoryOnly());
+  public static ChainStorageClient memoryOnlyClient(
+      final EventBus eventBus, final DiskUpdateChannel diskUpdateChannel) {
+    final ChainStorageClient client = new ChainStorageClient(diskUpdateChannel, eventBus);
     eventBus.register(client);
     return client;
   }
```

Comment:
Given this was just creating a `TransactionPrecommit.memoryOnly()` maybe it should just create a stub `DiskUpdateChannel` implementation that returns the same `DatabaseUpdateResult.successfulWithNothingPruned()` as the result of `onDiskUpdate` calls.